analysis/analysis_utils.py

```diff
@@ -227,7 +227,9 @@ def process_results(self,search,image_params,res_filepath,likelihood_level):
         print("Processing Results")
         print('---------------------------------------')
         while likelihood_limit is False:
+            print('Starting pooling...')
             pool = mp.Pool(processes=16)
+            print('Getting results...')
             results = search.get_results(res_num,chunk_size)
```

Comment:
Now if you put something other than `None` in results, it never gets used. Did you want that option to still work?

Reply:
No. I think that was originally an attempt to get a single process_results class for grid search and region search. I eventually gave up and coded a separate process_region_results() function.